### DISCUSSION TITLE
Hii

### DISCUSSION THREAD

**Discussion by puneethvk18** [community/community#20210] [Other Feature Feedback, Questions, & Ideas]
Hello everyone! 👋
I am Puneeth, an aspiring full stack developer and a tech enthusiast. Nice meeting you all. I will be looking forward to interact, collaborate and get along with you in future.

**Comment by radar-blip** [comment]
Greetings from USA.

**Comment by alexwholland** [comment]
Hello from Canada 🇨🇦

**Comment by isinicolle** [comment]
Hello from Latin America  🖐🏻 
📌(Specified Honduras - Central America)
